src/base/bittorrent/torrenthandle.h

```diff
@@ -333,7 +333,7 @@ namespace BitTorrent
         void setName(const QString &name);
         void setSequentialDownload(bool b);
         void toggleSequentialDownload();
-        void setFirstLastPiecePriority(bool enabled);
+        void setFirstLastPiecePriority(bool enabled, const QVector<int> &updatedFilePrio = {});
```

Comment:
It's a bad idea to make this changes in class interface.
You need to extract private part from this method.

Reply:
>You need to extract private part from this method.

Can you elaborate a bit?

Reply:
You need to keep `void setFirstLastPiecePriority(bool enabled);` here, add private `void setFirstLastPiecePriorityImpl(bool enabled, const QVector<int> &updatedFilePrio = {});` and then call it from `setFirstLastPiecePriority` and from `prioritizeFiles`.

Reply:
Thanks, done.